base/pkg/generate.jl

```diff
@@ -149,6 +150,17 @@ function travis(pkg::AbstractString; force::Bool=false)
         #script:
```

Comment:
I feel like more of these descriptions should be going in the package development documentation (based on feedback I've seen various places, people seem to be missing that it exists, or it needs to be expanded to cover more use cases) rather than comments in a generated file.

Also I guess I was hoping https://github.com/JuliaCI/travis-build/issues/1 would be resolved a little sooner, in that at least `coverage=true` could be made the default on Travis before we figure out the rest of it.